FeedStoreChallenge/Infrastructure/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Alexander on 17/5/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not meant to be subclassed so you can mark it as final:

```suggestion
final class ManagedFeedImage: NSManagedObject {
```